packages/tooltip/stories/tooltip.stories.ts

```diff
@@ -319,7 +319,8 @@ export const selfManaged = ({
             ?delayed=${delayed}
             open=${open}
         >
-            This is a tooltip.
+            Add paragraph text by dragging the Text tool on the canvas to use
+            this feature
```

Comment:
are we missing a full stop after feature?

Reply:
For now I adopted this from a product. But, you may be right that they should have a period, too. Particularly in relation to [this](https://spectrum.adobe.com/page/tooltip/#Be-concise) in the docs.  I'll cycle with them and then come back to this if their output changes.